RELEASE_INSTRUCTIONS.md

```diff
@@ -27,3 +27,5 @@ or a predetermined release date is approaching
   - [ ] [Deployment Plan](http://bamboo.corp.alleninstitute.org/deploy/viewDeploymentProjectEnvironments.action?id=164855841)
 - [ ] After release/deployment, merge release branch back into dev and delete the release branch
 - [ ] Announce release on https://community.brain-map.org
+
+*note: CircleCI will deploy the release version of ipfx from the master branch to pypi when the command "git tag v#.#.#" (e.g. "git tag v1.0.0") runs.
```

Comment:
Rather than adding this important step as a footnote, need to explain this in the bullet point before the point describing merging the release to the master branch.
Remove "push to pipy" since we are not manually pushing to pipy.
Also, after release is done we should probably merge master into dev rather than release into dev. This way if some hotfixes were merged into master between releases, then they would also be integrated into dev.

Also, we should probably chn

Reply:
revised as discussed. assuming the code flow would always be bugfix-branch(pr)=>dev=>release=>master